src/client/components/additionalConfigs/additionalConfigs.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faVolumeUp, faCog } from '@fortawesome/free-solid-svg-icons';
+
+import classes from './additionalConfigs.css';
+
+const volumeIcon = <FontAwesomeIcon icon={faVolumeUp} />;
+const cogIcon = <FontAwesomeIcon icon={faCog} />;
+
+class AdditionalConfigs extends Component {
```

Comment:
If you are not using state of component no need to use class component here
Functional component will be enough